backend/spec/features/admin/products/stock_management_spec.rb

```diff
@@ -42,46 +45,64 @@
     end
 
     it "can create a positive stock adjustment", js: true do
-      adjust_count_on_hand('14')
+      adjust_count_on_hand(variant.id, '14')
       stock_item.reload
       expect(stock_item.count_on_hand).to eq 24
       expect(stock_item.stock_movements.count).to eq 1
       expect(stock_item.stock_movements.first.quantity).to eq 14
     end
 
     it "can create a negative stock adjustment", js: true do
-      adjust_count_on_hand('-4')
+      adjust_count_on_hand(variant.id, '-4')
       stock_item.reload
       expect(stock_item.count_on_hand).to eq 6
       expect(stock_item.stock_movements.count).to eq 1
       expect(stock_item.stock_movements.first.quantity).to eq(-4)
     end
 
     it "can toggle backorderable", js: true do
-      toggle_backorderable(value: false)
+      toggle_backorderable(variant.id, value: false)
 
       click_link "Product Stock"
       within("tr#spree_variant_#{variant.id}") do
         expect(find(:css, "input[type='checkbox']")).not_to be_checked
       end
     end
 
-    def adjust_count_on_hand(count_on_hand)
-      within("tr#spree_variant_#{variant.id}") do
+    def adjust_count_on_hand(variant_id, count_on_hand)
+      within("tr#spree_variant_#{variant_id}") do
         find(:css, "input[type='number']").set(count_on_hand)
         click_icon :check
       end
       expect(page).to have_content('Updated Successfully')
     end
 
-    def toggle_backorderable(value: true)
-      within("tr#spree_variant_#{variant.id}") do
+    def toggle_backorderable(variant_id, value: true)
+      within("tr#spree_variant_#{variant_id}") do
         find(:css, "input[type='checkbox']").set(value)
         click_icon :check
       end
       expect(page).to have_content('Updated Successfully')
     end
 
+    context "with two variants, one of which tracks inventory while the other doesn't" do
+      it "allows modifying backorderable only on the variant which tracks inventory", js: true do
```

Comment:
I tried doing a hover here as well, like in the next example below, but for some reason Selenium/ChromeDriver doesn't pick up the tooltip on this one. Didn't seem worth the effort to solve this.